src/modules/ctc/src/ctc.cc

```diff
@@ -14,6 +14,21 @@
 namespace ctc
 {
 
+Ctc::Ctc(void)
+{
+}
+
+Ctc::Ctc(const types::TrackId track_id)
+{
+    if (track_id == types::TrackId::TRACKID_GREEN)
+    {
+        std::filesystem::path base_path = std::filesystem::current_path();
+        std::filesystem::path path      = base_path / ".." / "tests" / "common" / "test_csv" / "green_line_schedule.csv";
```

Comment:
We should handle the case when this file path doesn't exist, otherwise we can't run the application in any other location since we'll segfault and crash.

Reply:
implement else case with `std::abort()`. 
This constructor is only for ease of test integration to set up ctc quickly for green line. Later default constructor will be used to create empty instance of ctc, and dispatcher will click "Choose File" to populate either red line or green line blocks data and schedule. But currently backend doesn't support that so that expansion will be another ticket